oxidation/client/src/components/ProfileHeaderPopover.vue

```diff
@@ -0,0 +1,139 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-list class="container">
+    <ion-item class="container__item profile-email">
```

Comment:
Any reason to have `__` in the class name?

Reply:
This is a way of naming the css according to the type of element (Block, Element, Modifier)
[link: BEM nomenclature](https://alticreation.com/bem-pour-le-css/ )